webui/react/src/components/kit/LineChart.tsx

```diff
@@ -333,7 +333,7 @@ export const ChartGrid: React.FC<GroupProps> = React.memo(
       chartsProps.forEach((chart) => {
         chart.series.forEach((serie) => {
           Object.entries(serie.data).forEach(([xAxisOption, dataPoints]) => {
-            if (dataPoints.length > 0) {
+            if ((dataPoints?.length ?? 0) > 0) {
```

Comment:
Will any of the charts show data in this case? If not then it'd be better for the compare pain to use the `Empty` component to say that there are no metrics available than show a bunch of blank charts.

Reply:
If it's a momentary state then we should show loading state instead of "No Data". LineChart has loading state support already.

Reply:
ok I traced this back to how we use x axis in `useTrialMetrics` https://github.com/determined-ai/determined/pull/7122/commits/0575f29d2a14392b38eb78e69a8935859aaff1d6